frame/contracts/src/lib.rs

```diff
@@ -150,6 +150,12 @@ type RelaxedCodeVec<T> = WeakBoundedVec<u8, <T as Config>::MaxCodeLen>;
 type AccountIdLookupOf<T> = <<T as frame_system::Config>::Lookup as StaticLookup>::Source;
 type DebugBufferVec<T> = BoundedVec<u8, <T as Config>::MaxDebugBufferLen>;
 
+/// The old weight type.
+///
+/// This is a copy of the [`frame_support::weights::OldWeight`] type since the contracts pallet
+/// needs to support it indefinitely.
+type OldWeight = u64;
```

Comment:
it will be more clear to name it `OldWeightV1`

Reply:
I disagree: This sounds like there is a `V1` and `V2` of `OldWeight` which there is not.

Reply:
You are right/

Reply:
Please approve if you are happy so we can proceed.